nextmove/src/main/java/no/difi/meldingsutveksling/nextmove/DpiConversationResource.java

```diff
@@ -16,4 +19,27 @@
 
     private String title;
 
+    private DpiConversationResource(String conversationId,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS00107)